pennylane_lightning_gpu/src/bindings/Bindings.cpp

```diff
@@ -75,6 +75,9 @@ void StateVectorCudaManaged_class_bindings(py::module &m) {
     py::class_<StateVectorCudaManaged<PrecisionT>>(m, class_name.c_str())
         .def(py::init<std::size_t>())              // qubits, device
         .def(py::init<std::size_t, DevTag<int>>()) // qubits, dev-tag
+        .def(py::init([](const std::size_t num_qubits) {
+            return new StateVectorCudaManaged<PrecisionT>(num_qubits);
+        })) // qubits
```

Comment:
How does this constructor differ from line 76? Also, for pass-by-value, we shouldn't need to use `const`.

Reply:
no difference! let me remove it.